Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidSupport.cs

```diff
@@ -21,6 +21,11 @@ class AndroidSupport
 
         public static void Initialize()
         {
+            InputSystem.RegisterLayout<AndroidDevice>(
+                matches: new InputDeviceMatcher()
+                    .WithInterface(kAndroidInterface)
+                    .WithDeviceClass("AndroidDevice"));
```

Comment:
Would suggest replacing this "AndroidDevice" string with `nameof(AndroidDevice)`